README.md

```diff
@@ -1 +1,133 @@
-# webpack-exercise
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Todo List Project](#-todo-list-project)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Todo List Project<a name="about-project"></a>
+
+**Minimalist To Do List Project** 
+is a website that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. It is inspired by the minimalist website. That website is already offline but you can still play with thanks to web.archive.org.
+
+
+## 🛠 Built With <a name="built-with"></a>
+    HTML
+    CSS
+    Javascript(ES6)
+    Webpack
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Please use the following [link](https://sabaahmad786.github.io/Todo-list/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] [optional] @SabaAhmad786 great work on your readme, however, you forgot to add the Features section in your readme.